core/store.go

```diff
@@ -140,6 +145,23 @@ func DelByPtr(ptr unsafe.Pointer) bool {
 	return false
 }
 
+// List all keys in the store by given pattern
+func Keys(p string) []string {
+	storeMutex.RLock()
+	defer storeMutex.RUnlock()
+	keypoolMutex.RLock()
+	defer keypoolMutex.RUnlock()
+
+	keys := make([]string, 0)
+	for k := range keypool {
+		if found, err := path.Match(p, k); err == nil && found {
```

Comment:
I'm unsure if ignoring the error message here is a good idea. We should return the error to the caller if something bad happens